styles/home.scss

```diff
@@ -1,4 +1,7 @@
+body {
```

Comment:
You should probably move `global` selectors to another final like `base.scss` or something.